e2e/test/scenarios/visualizations-charts/line-bar-tooltips.cy.spec.js

```diff
@@ -598,7 +601,8 @@ describe("scenarios > visualizations > line/bar chart > tooltips", () => {
       });
     });
 
-    // TODO: this card should be converted properly, right now it isn't
+    // TODO: Fix series have different colors in visualizer because of different
```

Comment:
This only applies to "add series" cards being converted to "visualizer" cards, correct?